### PR TITLE
bugfix/revert wrongfully deleted files

### PR DESCRIPTION
**Pull request by odelame**

### CODE REVIEW

**Review by michelhe — APPROVED** [review]
Deleted by accident on #257 , right?

**Comment by Jongy** [comment]
Can you please create a PR in gProfiler that points to this commit, and ensure gProfiler builds & the tests pass? (then we can merge this & the gProfiler PR).

**Comment by grariel** [comment]
@odelame @michelhe what about this? It blocks @gAmitFrenkel

File: granulate_utils/linux/process.py [hunk 1]
```diff
@@ -171,3 +171,33 @@ def search_for_process(filter: Callable[[psutil.Process], bool]) -> Iterator[psu
         with contextlib.suppress(NoSuchProcess, AccessDenied):
             if is_process_running(proc) and filter(proc):
                 yield proc
+
+
+class ProcCgroupLine:
```
Comment (Jongy) [inline granulate_utils/linux/process.py:176]: What's this? I don't see it previously removed in https://github.com/Granulate/granulate-utils/pull/257.

**Review by Jongy — APPROVED** [review]

**Pull request merged**